aiozipkin/record.py

```diff
@@ -42,7 +46,7 @@ def name(self, n):
         return self
 
     def set_tag(self, key, value):
-        self._tags[key] = value
+        self._tags[key] = str(value)
```

Comment:
this line should be updated too in order to make tests green
https://github.com/aio-libs/aiozipkin/blob/73633be49975349ab02681dcfb0f7476a5bfdb48/tests/test_aiohttp_ingegration.py#L64

Reply:
fixed

Reply:
good catch, btw